contents/blog/b2b-saas-product-metrics.md

```diff
@@ -0,0 +1,210 @@
+---
+date: 2022-06-14
+title: The most useful B2B SaaS product metrics (and how to measure them)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/give-back-friday.png
+featuredImageType: full
+categories: ["Guides", "PostHog Academy", "Product analytics"]
+author: ["andy-vandervell"]
+---
+
+So, you're building a B2B SaaS product.
+
+Everyone knows you've got to measure stuff to succeed. This is not news. 
+
+What, why, and how? These are the questions that matter.
+
+In this guide we will:
+
+1. Reveal what makes a useful metric for a B2B product
+2. Explain some of the common metrics used by B2B SaaS companies
+3. Help you choose the right metrics for your product
+
+This guide is about _product_ metrics, rather than revenue or wider business metrics. Naturally, they're important too.
+
+Oh and, spoiler alert, we build an open-source analytics platform called PostHog. It's basically Mixpanel + Hotjar + Optimizely + LaunchDarkly in one tool you can host yourself. 
+
+In other words: it's the thing you should use to understand what your users are doing.
+
+Read our [product page](/product) and [documentation](/docs) if that sounds like something you need.
+
+**Contents:**
+
+- [What makes a good B2B product metric?](#what-makes-a-good-b2b-product-metric)
+- [Choosing a North Star metric](#choosing-a-north-star-metric)
+- [Common B2B product metrics](#common-b2b-saas-product-metrics)
+- [Recommended B2B product metrics](#recommended-b2b-product-metrics)
+- [Further reading](#further-reading)
+
+> This article is part of our [PostHog Academy series](/blog/categories/posthog-academy) where we explain the fundamentals of product analytics.
+
+## What makes a good B2B product metric?
+
+Good metrics are understandable, comparative, specific and actionable.
+
+Here's what those mean in a little more detail:
+
+1. **Understandable:** Can you easily explain a metric to anyone in the business? If you can't, it's unlikely people will want to use or refer to it.
+
+2. **Comparative:** Can you benchmark a metric against your industry, or compare internally using cohorts?
+
+3. **Specific:** Some broad trends are useful, but we recommend metrics that measure specific actions. Don't measure how long users use your product, measure how long it takes for them to get value from it.
+
+4. **Actionable:** Can you easily identify actions to take when you see a drop in a metric? If yes, you're onto a winner. Specific metrics are the most actionable metrics, too.
+
+## Choosing a North Star metric
```

Comment:
I wonder if the North Star section should be cut or maybe form part of 'further reading' under 'common B2B SaaS metrics'? It's a bit weird to have a guide and then say 'oh you now need to read another guide' and the section as a whole doesn't add anything.

Reply:
Hmm, yeah, this was orginally going to be a boxout. Agree it rather disrupts the flow. Might revert back to a boxout, or just remove entirely.